lib/git/tail/runner.rb

```diff
@@ -72,15 +72,13 @@ def clean_local(branch)
         # make sure it's only in one.
         new_log_entries.pop if new_log_entries.last == old_log_entries.first
 
-        if old_log_entries.empty?
+        if old_log_entries.count <= 1
           out :detail, 2, "No commits prior to cutoff date. Skipping."
```

Comment:
This message should be updated to accurately reflect the change. Otherwise, looks good.